packages/react/src/components/CodeSnippet/CodeSnippet.js

```diff
@@ -226,10 +226,14 @@ function CodeSnippet({
     <div {...rest} className={codeSnippetClasses}>
       <div
         ref={codeContainerRef}
-        role={type === 'single' ? 'textbox' : null}
-        tabIndex={type === 'single' && !disabled ? 0 : null}
+        role={type === 'single' || type === 'multi' ? 'textbox' : null}
+        tabIndex={
+          type === 'single' || (type === 'multi' && !disabled) ? 0 : null
```

Comment:
should this be the following instead? looks like the original condition for `single` is changing
```suggestion
          (type === 'single' || type === 'multi') && !disabled ? 0 : null
```

Reply:
It must have auto formatted it. I think if we use your suggestion it _won't_ next time. I'll work on getting that added!